src/ipc/ipc.cpp

```diff
@@ -265,6 +268,7 @@ double compute_collision_free_stepsize(
     const Eigen::MatrixXd& V0,
     const Eigen::MatrixXd& V1,
     const BroadPhaseMethod method,
+    const double min_distance,
```

Comment:
use `min_distance/2` for inflation radius in `construct_collision_candidates` call